### DISCUSSION TITLE
How to use an if statement in custom labels

### DISCUSSION THREAD

**Discussion by AlexanderLeporiday** [cgiesche/streamdeck-homeassistant#269] [Help and Support]
I have a custom label that returns {{ app_id }} which is something like "com.google.android.tvlauncher" but I want to display this as "Home" and "com.plexapp.android" as "Plex".

I can't figure out how to do an if statement in the custom label field that would do the trick. So far everything I have tried just breaks the button until I remove it. Basically I have button that switch my TV inputs but I want the label to display what app is open of if "TV" is showing. TV will be any source/app that isn't a known app.

Any ideas?

**Comment by AlexTu2** [comment]
Did you figure this out?

I tried doing something like

`{% if {{state}} == 'on' %}
    {{ 'close' }}
{% else %}
    {{ 'on' }}
{% endif %}`

but it didn't work either

**Reply by DAveShillito** [reply]
I too would like some clarity on this.
I am trying to map {{is_volume_muted}} of true/false to muted/unmuted for my TV
